content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,21 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI container
+runtime.
+
+The OCI container runtime requirements are as follows:
+
+ * [crun][crun]: version 1.9 or greater (1.13+ recommended)
+
+Please note that [runc][runc] is the default OCI container runtime in several
+Kubernetes distributions. Version `1.1.z` of runc doesn't support all the
+features needed by the Kubernetes implementation. If there is a newer release of
+runc than 1.1 available for use, check its documentation and release notes for
+compatibility (look for idmap mounts support in particular, as that is the
+missing feature).
+
+The requirements for the 
```

Comment:
```suggestion
To use user namespaces with Kubernetes, you also need to use a CRI
```

Reply:
Didn't do the CRI change, so I didn't apply this.

Reply:
I'm recommending the clarification around the two kind of container runtimes to help readers; for beta quality, we want to reduce the chance of a reader feeling confused (for alpha, we're less bothered).

Reply:
Can you read the changes I pushed and comment on that, if this is still not clear?

Reply:
(sure, I can confirm I was reviewing based on the state of the PR at the time I commented; https://github.com/kubernetes/website/pull/45178/files#r1523158216 was considering the commits at that time)

Reply:
This change: https://github.com/kubernetes/website/pull/45178#discussion_r1522133388

If it's a bad idea to make it and will harm readers' understanding, let's not do it. Otherwise, I'm asking for that change as a docs approver. Please consider accepting it.

Reply:
Applied.